packages/webdriver-utils/src/metadata/normalizeData.js

```diff
@@ -0,0 +1,34 @@
+export default class NormalizeData {
+  osRollUp(os) {
+    if (os.toLowerCase().startsWith('win')) {
+      return 'Windows';
+    } else if (os.toLowerCase().startsWith('mac')) {
+      return 'OS X';
+    } else if (os.toLowerCase().includes('iphone') || os.toLowerCase().startsWith('ios')) {
+      return 'iOS';
+    } else if (os.toLowerCase().startsWith('android')) {
+      return 'Android';
+    }
+    return os;
+  }
+
+  browserRollUp(browserName, device) {
+    if (device) {
+      if (browserName?.toLowerCase().includes('chrome')) {
+        return 'chrome';
+      } else if ((browserName?.toLowerCase().includes('iphone') ||
+          browserName?.toLowerCase().includes('ipad'))) {
+        return 'safari';
+      }
+    }
+    return browserName;
+  }
+
+  browserVersionRollUp(browserVersion, deviceName, device) {
+    if (device) {
+      // return `${this.osRollUp(os)} ${osVersion?.split('.')[0]}`;
+      return deviceName;
+    }
+    return browserVersion?.split('.')[0];
+  }
```

Comment:
Didnt understand why are we using deviceName for browserVersionRollUp ?

Reply:
Reason being that in case of devices we don't really get the `browserVersion` properly. In capabilities we are getting the `browserVersion` as the `iphone` or `device_name` itself in certain devices. Responses differ when using different appium and selenium Version. And from the UI perspective to keep it consistent we moved it to keep `device_name` this ensures no UI change is required and nothing is breaking.

Reply:
Got it, then can we just rename function to `browserVersionOrDeviceNameRollup` and add the comment on top ? [ the same comment as what you have explained above